### DISCUSSION TITLE
Request a Refund

### DISCUSSION THREAD

**Discussion by smc-jp** [aclap-dev/video-downloadhelper#1398] [License & Purchase requests]
### Your question

I recently purchased the Firefox version by mistake, intending to use it on Chrome, and am therefore unable to access the service. Could you kindly assist with a refund for the initial purchase? Once refunded, I plan to complete the correct purchase for the Chrome version. Thank you for your attention to this matter.

Invoice ID	00491716

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: ko
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"key":"1fc080ee518040ba","mismatch":true,"other_browser":"Firefox","this_browser":"Chrome"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Whale/3.28.266.14 Safari/537.36
{
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": true,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 22,
    "successfull_dl": 3
}

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
